src/ILLink.RoslynAnalyzer/ILLink.RoslynAnalyzer.csproj

```diff
@@ -16,6 +16,10 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisVersion)" PrivateAssets="all" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\external\corert\**\*.cs" />
```

Comment:
Why do we need the corert module?

Reply:
I remember needing it for something at some point, but building without it passes, so I'll remove it.